packages/jmix-front-generator/src/generators/react-typescript/app/template/i18n-message-packs/zh-cn.json

```diff
@@ -10,6 +10,7 @@
   "common.requestFailed": "请求的操作执行失败",
   "common.retry": "重试",
   "common.back": "common.back [TRANSLATION NOT AVAILABLE]",
+  "common.clone": "克隆",
```

Comment:
We don't add messages in foreign languages, if not sure about the translation. Better use "common.clone [TRANSLATION NOT AVAILABLE]" as value. The same in other languages.